samples/SkiaJvmSample/src/main/kotlin/SkiaJvmSample/App.kt

```diff
@@ -120,3 +116,17 @@ fun createWindow(title: String, exitOnClose: Boolean) = SwingUtilities.invokeLat
     skiaLayer.paint(window.graphics)
     window.isVisible = true
 }
+
+private fun parseArgs(args: Array<String>): Int {
```

Comment:
Why not property as everything else?

Reply:
I was thinking about passing the number of windows parameter via the command line, something like `./gradlew <run task> <window count>` so I had 2 options:
 - `./gradlew run -Dskiko.example.window.count=2`
 - `./gradlew run --args='2'`
 the last one is much shorter.